Backpack/Calendar/Classes/BPKCalendarCell.m

```diff
@@ -184,10 +185,19 @@ - (void)configureAppearance {
     UIColor *rangeTitleColor = [self colorForCurrentStateInDictionary:self.appearance.titleColors];
 
     if (self.titleLabel.text) {
+        if (self.wholeMonthSelection) {
+            self.titleLabel.attributedText = [BPKFont attributedStringWithFontStyle:BPKFontStyleTextHeading5
+                                                                            content:self.titleLabel.text
+                                                                          textColor:rangeTitleColor];
+            [self setSelected:NO];
```

Comment:
I assume this is to hide the dark blue selection circle, could we add a comment to indicate that is the need for this line of code?